src/coreclr/dlls/mscorrc/mscorrc.rc

```diff
@@ -162,6 +162,7 @@ BEGIN
     IDS_EE_NDIRECT_DISABLEDMARSHAL_LCID     "The LCIDConversionAttribute is not supported when runtime marshalling is disabled."
     IDS_EE_NDIRECT_DISABLEDMARSHAL_PRESERVESIG "Setting PreserveSig to false for a P/Invoke is not supported when runtime marshalling is disabled."
     IDS_EE_NDIRECT_DISABLEDMARSHAL_VARARGS  "Using a variable argument list in a P/Invoke is not supported when runtime marshalling is disabled."
+    BFA_ASYNC_PINVOKE_METHOD                "A PInvoke method cannot be marked as async."
```

Comment:
All BFA_.... are in a block together. Can this be part of that block? (Alternatively, do not call it BFA...)